pkg/insightstore/report.go

```diff
@@ -0,0 +1,281 @@
+// Copyright 2020 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package insightstore
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/pipe-cd/pipe/pkg/model"
+)
+
+var ErrValueNotFound = errors.New("value not found")
+
+// deploy frequency
+
+// DeployFrequencyReport satisfy the interface `Report`.
+type DeployFrequencyReport struct {
+	AccumulatedTo int64                    `json:"accumulated_to"`
+	Datapoints    DeployFrequencyDataPoint `json:"datapoints"`
```

Comment:
Could you unify this word? 
`DataPoint` or `Datapoint` for all.
(Currently, in the model package, we are using `DataPoint`.)